app/javascript/multi-stamp/components/MultiStamp.vue

```diff
@@ -127,6 +127,10 @@ export default {
 
     // Default volume to define in the UI for the volume control
     defaultVolume: { type: String, required: false, default: null },
+
+    // Acceptable plate purpose names that can be used as source plates e.g. "['PurposeA', 'PurposeB']"
+    // See computed method for conversion to array
+    acceptablePurposes: { type: String, required: false, default: '[]' },
```

Comment:
Do they have to be passed in as strings?

Reply:
Is this `MultiStamp` component only used in situations where plates are scanned in? Or can it also be used for scanning tubes in? Think the former, because we've also got a component called `MultiStampTubes`?

Just checking because this new code assumes we're scanning plates, I think.

Reply:
Yeah they are passed as string interpolations in the erb that passes the variable values to the Vue JS component.
See `app/views/plate_creation/multi_stamp.html.erb`

I believe it's just used for plates, as it has many references to plate in the template and methods. The tube version also calls a separate file for its scanValidations.